src/packages/cli/src/Init.ts

```diff
@@ -49,6 +49,29 @@ export const defaultEnv = (
   return env
 }
 
+export const defaultPort = (provider = 'postgresql') => {
+  switch (provider) {
+    case 'mysql':
+      return 3306
+    case 'sqlserver':
+      return 1433
+    case 'postgresql':
+    default:
+      return 5432
+  }
+}
+
+export const defaultURL = (
+  provider = 'postgresql',
+  port = defaultPort(provider),
+  schema = 'public',
+) => {
+  const schemaExists = provider === 'postgresql'
+  return `${provider}://johndoe:randompassword@localhost:${port}/mydb${
+    schemaExists ? `?schema=${schema}` : ''
+  }`
+}
```

Comment:
We could also add a `sqlite` provider here to be complete I think, default could be `file:./dev.db`

https://www.prisma.io/docs/concepts/database-connectors/sqlite

Reply:
Silly me. Thanks. I've responded to your comments btw.